samples/react-chat/src/App.tsx

```diff
@@ -1,7 +1,8 @@
-import React, { memo, useCallback, useState } from 'react';
+import React, { memo, useCallback, useState, useEffect } from 'react';
```

Comment:
it doesn't look like useEffect is used in this file

Reply:
Correct, removed.